src/app/resync/login/page.module.scss

```
@@ -5,6 +5,10 @@
 
 .container {
   @include utilities.funnel-page-spacing;
+  // iOS WebView 에서 키보드가 올라올 때 비밀번호 입력칸이 가려지지 않도록
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)